WcaOnRails/app/assets/stylesheets/navbar-static-top.scss

```diff
@@ -1,6 +1,6 @@
 html,
 body {
-  font-size: 1em;
+  font-size: 14px;
```

Comment:
As discussed offline, lets remove this entirely in favor of bootstrap's styling.